pyscf/tools/get_xyz.py

```diff
@@ -0,0 +1,8 @@
+import re
+
+def get_xyz(db_data:str, name:str)-> str:
+    p = re.compile(f'(NAME = {name})(.*?)Date modified:(.*?)\)(.*?)NAME', flags=re.DOTALL)
```

Comment:
Is it 100% necessary to start from the `Date modified` part? There's no way to just treat the lines starting with `#` as "comments" (i.e. ignore those lines completely) and just import the lines that don't start with #?

The comments are meant contain "optional" information. Some geometry files might not have a "Date modified". Especially files that have never been modified, if they just say "Date added". 

It seems like searching for "Date modified" and parsing the next line, opens the door for many potential bugs in case the "Date modified" line is not there, or is spelled incorrectly, or is present but in the wrong place.

If it's not possible to treat lines starting with # as comments (i.e. parsing only the lines that don't start with #), then let's just put a # after the "last" comment line, and have the parser read everything between the last # and the next `NAME`.

Reply:
What I am doing is string regex, which is basically looking for patterns in strings. 
So we need some kind of identifier to extract text.

Reply:
Also, I just learnt that pyscf can understand that lines starting with a # are comments. So we can read all the lines between two names by a simplified regex like this:

`p = re.compile(f'(NAME = {name})(.*?)NAME', flags=re.DOTALL)`